spring-cloud-aws-autoconfigure/src/test/java/io/awspring/cloud/autoconfigure/s3/S3AutoConfigurationTests.java

```diff
@@ -114,9 +130,17 @@ void createsStandardClientWhenCrossRegionModuleIsNotInClasspath() {
 
 	@Nested
 	class OutputStreamProviderTests {
+
```

Comment:
Could you add test under  `S3TemplateAutoConfigurationTests` if someone defined `S3TransferManager` as a bean.

Reply:
Added